<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ConverterTab">
     <attribute name="title">
      <string>Конвертация</string>
     </attribute>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>331</width>
        <height>231</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Выбор системы исчисления числа конвертации</string>
       </property>
      </widget>
      <widget class="QComboBox" name="inputComboBox">
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="inputLineEdit"/>
      <widget class="QPushButton" name="translateButton">
       <property name="text">
        <string>Конвертировать</string>
       </property>
      </widget>
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>Ввод системы исчисления в которую выводить</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="outputBaseLineEdit"/>
      <widget class="QLabel" name="outputLabel_2">
       <property name="text">
        <string>Вывод системы исчисления</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="outputLineEdit"/>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>460</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SettingsTab">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="labelColorButton">
         <property name="text">
          <string>Выбор цвета текста</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="saveSettingsCheckBox">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConvert_Bases_App">
    <property name="title">
     <string>Convert Bases App</string>
    </property>
   </widget>
   <addaction name="menuConvert_Bases_App"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
